src/SonarLint.VSSpecificAssemblies.props

```diff
@@ -143,4 +143,16 @@
     <StrongNamerKeyFile>$(AssemblyOriginatorKeyFile)</StrongNamerKeyFile>
   </PropertyGroup>
 
+
+  <!-- ******************************************************************************* -->
+  <!-- VCProject assembly -->
+  <!-- ******************************************************************************* -->
+  <ItemGroup Label="VCProjectEngine" Condition="$(RequiresVCProjectEngine)=='true'">
+    <Reference Include="Microsoft.VisualStudio.VCProjectEngine">
+      <HintPath>$(ThirdPartyPath)\Microsoft.VisualStudio.VCProjectEngine.dll</HintPath>
+      <Private>$(CopyVSAssembliesToOutput)</Private>
+    </Reference>
```

Comment:
Same pattern as for the TeamExplorer assemblies - three versions of the assembly are checked in, and we pick the one to build against depending on the version of VS being targetted.